### DISCUSSION TITLE
表格嵌套时，2.9.9版子表格列不对齐

### DISCUSSION THREAD

**Discussion by ylens** [layui/layui#1860] [问答]
在使用layui最新版本2.9.9版后，发现表格嵌套时，子表格的表头与表的列不对。而2.9.8版则无此现象；
![文件1](https://github.com/layui/layui/assets/47489538/db89acb0-bd81-40a9-ab95-5f6da9ccfb75)
改回2.9.8版，此问题消失
![文件2](https://github.com/layui/layui/assets/47489538/a6d8c884-2ce9-4716-81c5-577ef221d669)

这要怎么解决？

**Comment by sentsim** [comment]
需要提供相关的业务代码

**Comment by suzehong** [comment]
请问有解决吗？
